packages/admin/admin-icons/icons/non-breaking-space.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <path d="M14.067 13.5H15v-11h-.933v11ZM1 13.5h.933v-11H1v11Zm7.73-8.25-.002 2.089 2.072.001v1.32H8.728l.001 2.09H7.271l-.002-2.09H5.2V7.34l2.07-.001V5.25h1.46Z"/>
+</svg>
```

Comment:
New icon is coming in https://github.com/vivid-planet/comet/pull/1348

Reply:
Done here: ff9e53f005287597dd56fc244276063bfeb7ad24